gradle/libs.versions.toml

```diff
@@ -19,8 +19,14 @@ androidx-camera-camera2 = { module = "androidx.camera:camera-camera2", version.r
 androidx-camera-compose = { module = "androidx.camera:camera-compose", version.ref = "camerax" }
 androidx-camera-lifecycle = { module = "androidx.camera:camera-lifecycle", version.ref = "camerax" }
 
+compose-material3 = { module = "org.jetbrains.compose.material3:material3", version = "1.9.0" }
```

Comment:
can be added new version `material3=1.9.0` and added into `compose-material3` and `compose-material3-window-size`